packages/rum/src/index.d.ts

```diff
@@ -53,8 +54,9 @@ declare module '@elastic/apm-rum' {
     captureError(error: Error | string): void
     addFilter(fn: FilterFn): void
   }
-  export { init, ApmBase as apm, ApmBase as apmBase, ApmBase }
+  const apmBase: ApmBase
```

Comment:
I wonder why this wasn't caught in our tests. Before this commit, we exported `ApmBase as apm`, which would be the class itself instead of its instance and shouldn't have the exposed APIs that we're testing for.

Reply:
Because we dont expect users to do `apm.serviceFactory` and I was about to catch this only because we were mocking ApmServer factory in our tests.